package.json

```
@@ -132,6 +132,7 @@
   "dependencies": {
     "@babel/runtime": "^7.23.7",
     "cookies": "^0.8.0",
+    "firebaseui": "^6.1.0",
```

This isn't a dependency of the main package, just the example app, so let's remove this.
```suggestion
    "firebaseui": "^6.1.0",
```